frontend/tests/README.md

```diff
@@ -129,3 +128,103 @@ All the E2E test commands can be found in `frontend/Makefile` and `frontend/pack
       1. For more information, visit [Playwright Inspector](https://playwright.dev/docs/debug)
 
       ![Playwright Inspector](https://user-images.githubusercontent.com/6309723/196775284-81bd4853-b4c6-45c4-827b-22e97b97ee06.png)
+
+---
+
+# Test/Env Differential
+
+Where tests are skipped vs. run in different environments.
+
+<table >
+<colgroup>
+<col >
+<col >
+<col >
+<col >
+<col >
+<col >
+<col >
+<col >
+</colgroup>
+<thead>
+  <tr>
+    <th></th>
+    <th colspan="7">TEST_ENV (defaults to local)</th>
```

Comment:
Great work on this table, this will be really useful to have handy

What do you think of also adding a blurb below or up-top briefly summarizing why certain tests are Dev/Staging Only or Deployed Env only? I think it'd be useful for anyone adding/debugging new tests who is unfamiliar

Reply:
I think it makes more sense to have those at the test level and not here, but maybe a brief summary on potential reasons would work here